bazel/repository_locations.bzl

```diff
@@ -274,6 +274,19 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2021-06-07",
         cpe = "cpe:2.3:a:grpc:grpc:*",
     ),
+    com_github_unicode_org_icu = dict(
+        project_name = "ICU Library",
+        project_desc = "Development files for International Components for Unicode",
+        project_url = "https://github.com/unicode-org/icu",
+        version = "70-1",
+        sha256 = "f30d670bdc03ba999638a2d2511952ab94adf204d0e14898666f2e0cacb7fef1",
+        strip_prefix = "icu-release-{version}",
+        urls = ["https://github.com/unicode-org/icu/archive/release-{version}.tar.gz"],
+        use_category = ["dataplane_ext"],
```

Comment:
Yeah, I think this would be desirable. It can be a separate PR as a followup, but we should ideally be able to better track which deps are used by core, extensions and contrib, since there are different security implications in the threat model. CC @envoyproxy/dependency-shepherds